src/toggle-switch/toggle-switch.scss

```diff
@@ -2,194 +2,208 @@
 // See LICENSE.md in the project root for license terms and full copyright notice.
 @import '../style/index';
 
-@mixin iui-toggle-switch {
+@mixin iui-toggle-switch-wrapper {
   @include iui-reset;
   display: flex;
   align-items: center;
+  gap: $iui-s;
+  align-items: center;
   font-size: $iui-font-size;
   width: fit-content;
-  padding: ($iui-baseline/2) 0;
   user-select: none;
   cursor: pointer;
-  position: relative;
+  isolation: isolate;
   @include themed {
     color: rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-2));
   }
 
+  // Disabled
+  &.iui-disabled {
+    cursor: not-allowed;
+    @include themed {
+      color: t(iui-text-color-muted);
+    }
+  }
+
+  // #region IE solution
```

Comment:
If it is not working on IE, maybe we can just remove these workaround?

Reply:
It's not displaying as a toggle switch in IE but it does still work (as a checkbox).  This CSS here makes it look ok in IE as a checkbox.